apps/foundation-demo/src/app/core/footer/footer.component.spec.ts

```diff
@@ -38,4 +42,12 @@ describe('FooterComponent', () => {
       expect(container.componentInstance.size).toEqual('medium');
     });
   });
+
+  describe('on all <daff-list>', () => {
```

Comment:
I think this is a poor test. Let's reimagine this driven from a requirement:

```
describe('footer')...
describe('the list of navigational links')

it('should render three daff-lists of type `link`')
```

Reply:
Resolved in df1d786